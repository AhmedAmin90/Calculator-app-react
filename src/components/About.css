.hero-section-text {
  width: 75%;
  margin: 100px auto;
  padding: 20px;
  display: flex;
  flex-direction: column;
  align-items: center;
  background-color: #f3efea;
  border: 2px #172b4d solid;
  box-shadow: 57px 43px 0 17px rgba(235, 196, 72, 0.78);
}

.hero-section-text h1 {
  font-size: 40px;
  font-weight: 700;
  color: #172b4d;
}

.my-description {
  color: #172b4d;
  font-size: 20px;
  height: 98px;
}

.connect-text {
  font-size: 16px;
  color: #172b4d;
}

.social-icons {
  color: #505f79;
  font-size: 16px;
  list-style: none;
  display: flex;
  width: 120px;
  justify-content: space-between;
  padding: 0;
}

.social-icons li a {
  color: #505f79;
}

.social-icons li a:hover {
  cursor: pointer;
  color: #172b4d;
}
