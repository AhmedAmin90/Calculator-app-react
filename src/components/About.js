import React from 'react';
import './About.css';

const About = () => (
  <div className="About">
    <div className="hero-section-text">
      <h1>
        AHMED AMIM
        <br />
        {' '}
        WEB DEVELOPER
      </h1>
      <p className="my-description">
        I’m a software developer! I can help you build a product
        , feature or website Look through some of my work and experience!
        If you like what you see and have a project you need coded, don’t hestiate to contact me.
      </p>
      <p className="connect-text"> lets Connect</p>
      <ul className="social-icons">
        <li>
          {' '}
          <a href="https://twitter.com/AhmedAmin12383" target="_blank" rel="noreferrer">
            <i className="fab fa-twitter" />
            {' '}
            {' '}
          </a>
          {' '}
        </li>
        <li>
          {' '}
          <a href="https://www.linkedin.com/in/web-developer/" target="_blank" rel="noreferrer">
            {' '}
            <i className="fab fa-linkedin-in" />
            {' '}
          </a>
          {' '}
        </li>
        <li>
          {' '}
          <a href="https://github.com/AhmedAmin90" target="_blank" rel="noreferrer">
            {' '}
            <i className="fab fa-github" />
            {' '}
          </a>
        </li>
        <li>
          {' '}
          <a href="https://angel.co/u/ahmed-amin-22" target="_blank" rel="noreferrer">
            {' '}
            <i className="fab fa-angellist" />
            {' '}
          </a>
        </li>
        <li>
          {' '}
          <a href="https://www.facebook.com/ahmed.amin.7564" target="_blank" rel="noreferrer">
            {' '}
            <i className="fab fa-facebook-f" />
            {' '}
          </a>
        </li>
      </ul>
    </div>
  </div>
);

export default About;
