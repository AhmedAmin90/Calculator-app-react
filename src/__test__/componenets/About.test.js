import React from 'react';
import { create } from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import About from '../../components/About';



it('Should renders correctly', () => {
    const elem = create(<About />).toJSON();
    expect(elem).toMatchSnapshot();
  });

test('displays heading', () => {
  render(<About />);
  const heading = screen.getByText('AHMED AMIM WEB DEVELOPER');
  expect(heading).toBeInTheDocument;
});
