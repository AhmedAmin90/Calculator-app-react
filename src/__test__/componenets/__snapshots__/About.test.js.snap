exports[`test Should renders correctly 1`] = `
<div
  className="About">
  <div
    className="hero-section-text">
    <h1>
      AHMED AMIM
      <br />
       
      WEB DEVELOPER
    </h1>
    <p
      className="my-description">
      I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
    </p>
    <p
      className="connect-text">
       lets Connect
    </p>
    <ul
      className="social-icons">
      <li>
         
        <a
          href="https://twitter.com/AhmedAmin12383"
          rel="noreferrer"
          target="_blank">
          <i
            className="fab fa-twitter" />
           
           
        </a>
         
      </li>
      <li>
         
        <a
          href="https://www.linkedin.com/in/web-developer/"
          rel="noreferrer"
          target="_blank">
           
          <i
            className="fab fa-linkedin-in" />
           
        </a>
         
      </li>
      <li>
         
        <a
          href="https://github.com/AhmedAmin90"
          rel="noreferrer"
          target="_blank">
           
          <i
            className="fab fa-github" />
           
        </a>
      </li>
      <li>
         
        <a
          href="https://angel.co/u/ahmed-amin-22"
          rel="noreferrer"
          target="_blank">
           
          <i
            className="fab fa-angellist" />
           
        </a>
      </li>
      <li>
         
        <a
          href="https://www.facebook.com/ahmed.amin.7564"
          rel="noreferrer"
          target="_blank">
           
          <i
            className="fab fa-facebook-f" />
           
        </a>
      </li>
    </ul>
  </div>
</div>
`;
